openfl-tutorials/interactive_api/MXNet_landmarks/README.md

```diff
@@ -0,0 +1,110 @@
+# MXNet Facial Keypoints Detection tutorial
+***Note: Please pay attention that this task uses the dataset from Kaggle. To get the dataset you
+will need a Kaggle account and accept "Facial Keypoints Detection" competition rules.***
+
+This tutorial shows how to use any other framework, different from already supported PyTorch and TensorFlow, together with OpenFl.
+
+## Installation of Kaggle API credentials
+
+**Before the start please make sure that you installed sd_requirements.txt on your virtual
+environment on an envoy machine.**
+
+To use the [Kaggle API](https://github.com/Kaggle/kaggle-api), sign up for
+a [Kaggle account](https://www.kaggle.com). Then go to the `'Account'` tab of your user
+profile `(https://www.kaggle.com/<username>/account)` and select `'Create API Token'`. This will
+trigger the download of `kaggle.json`, a file containing your API credentials. Place this file in
+the location `cd ~/.kaggle/kaggle.json`
+
+**Note: you will need to accept competition rules
+at** https://www.kaggle.com/c/facial-keypoints-detection/rules
```

Comment:
Suggestion for note section: 
```suggestion
---
**NOTE:**

You will need to accept [competition rules](https://www.kaggle.com/c/facial-keypoints-detection/rules).

---
```

Reply:
@ViktoriiaRomanova 
Could you please try to format all jupyter code by `Jupyterlab Code Formatter`? [Here](https://jupyterlab-code-formatter.readthedocs.io/en/latest/) is a short steps to install this tool. This tool availible only in jupyter-lab, so `JupyterLab Code Formatter` adds a button to Jupyter Lab to automatically format the code in notebooks. 


I guess below are the commands to be done according to the  `Jupyterlab Code Formatter` documentation:
```console
pip install jupyterlab
pip install jupyterlab_code_formatter
pip install black isort
```
Then, restart jupyter-Lab and use special button to automatically format the code in the notebook